src/virtual_ptdf_calculations.jl

```diff
@@ -174,50 +284,40 @@ function Base.isempty(vptdf::VirtualPTDF)
     !isempty(vptdf.temp_data) && return false
     return true
 end
-# Size related overload will work on the BA matrix
+
+"""
+Gives the size of the whole PTDF matrix, not the number of rows stored.
+"""
 Base.size(vptdf::VirtualPTDF) = size(vptdf.BA)
+"""
+Gives the cartesian indexes of the PTDF matrix (same as the BA one).
+"""
 Base.eachindex(vptdf::VirtualPTDF) = CartesianIndices(size(vptdf.BA))
 
 if isdefined(Base, :print_array) # 0.7 and later
     Base.print_array(io::IO, X::VirtualPTDF) = "VirtualPTDF"
 end
 
-# Get indices
-function _get_line_index(vptdf::VirtualPTDF, row::String)
-    row_ = findall(x -> x == row, vptdf.axes[1])
-    if length(row_) > 1
-        error("multiple lines with the same name $row in vptdf.axes[1]")
-    elseif length(row_) > 1
-        error("no line with name $row in vptdf.axes[1]")
-    else
-        row = row_[1]
-    end
-    return row
-end
-
-function _get_value(vptdf::VirtualPTDF, row::Int)
-    # get stored value if present in the dictionarr
-    if branch in keys(vptdf.data)
-        return vptdf.data[2][row]
-    end
-end
-
-function Base.getindex(vptdf::VirtualPTDF, row, column)
-    row_, column_ = to_index(vptdf, row, column)
-    return _getindex(vptdf, row_, column_)
-end
+"""
+Funciton for internal use only.
 
-# Define for ambiguity resolution
-function Base.getindex(vptdf::VirtualPTDF, row::Integer, column::Integer)
-    return _getindex(vptdf, row, column)
-end
+Gets the value of the element of the PTDF matrix given the row and column indices
```

Comment:
This is an internal function that doesn't require a docstring. Just documenting `Base.getindex` is enough